perfrunbook/system-load-and-compute-headroom.md

```diff
@@ -0,0 +1,120 @@
+# System Load and Compute Headroom
+
+From an economical perspective compute resources should be utilized fully.
+Ideally the system's performance should increase linearly up to 100% CPU load.
+This can be in the context of pure compute or peak load and fail-over resilience.
+Graviton instances and x86-type instances, such as c6i, behave fundamentally different under load.
+Any initial x86 advantage over Graviton starts to disappear beyond 50% system load.
+This effect is caused by symmetric multithreading and the Linux kernel's preferrence of CPU cores over CPU threads.
+In the following this behavior is investigated with a few illustrative workloads.
+
+### Experimental Setup
+
+The test systems are limited to eight cores to limit the maximum packet load needed to saturate them. 
+This is to exclude the influence of the networking infrastructure on the experiment.
+All systems run Amazon Linux 2 with Linux kernel 5.10.118.
+The test system are virtualized instances that use S3 networked storage.
+
+
+#### First experiment
+
+The systems to be tested are an 8 vCPU Intel instance (8375C, 3.5GHz), c6i.2xlarge, and an 8 CPU Graviton 3 instance, c7g.2xlarge.
```

Comment:
Don't need to include the x86 part number, just the instance type is sufficient here.